frontend/src/util/validations.js

```diff
@@ -0,0 +1,19 @@
+const _ = require("lodash");
+
+function languageValidator(language) {
+  return _.includes(["hebrew", "english"], language);
+}
+
+function paramsValidator(entitiyName) {
+  return _.includes(["dish", "list"], entitiyName);
+}
+
+function foreignKeysValidator(entity) {
+  return _.includes(["branch", ""], entity)
```

Comment:
Extract the array you have here to a const with a meaningful name (e.g `const ENTITIES_WITH_FOREIGN_KEYS_PARAMS` or something better, and also rename the function to the `isX` pattern, since it's boolean

Reply:
Also, why do you have an empty string there?